class throwsexception {
    public static void main(String[] args) throws ArithmeticException {
        System.out.println("inside main");
        int I = 0;
        int j = 100 / I;
        System.out.println("this statement is not printed");
    }
}


